__tests__/apiPostArticleComment.test.js

```diff
@@ -0,0 +1,79 @@
+const request = require("supertest");
+const app = require("../index");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+const jestSorted = require("jest-sorted");
+
+afterAll(() => {
+  db.end();
+});
+
+beforeEach(() => {
+  return seed(data);
+});
+
+describe("POST /api/articles/:article_id/comments", () => {
+  it("checking the body of returned comment", async () => {
+    const commentBody = "small cat";
+    const username = "lurker";
+    const articleId = 4;
+
+    const response = await request(app)
+      .post(`/api/articles/${articleId}/comments`)
+      .send({ username: username, body: commentBody });
+
+    expect(response.status).toEqual(201);
+    expect(response.body).toBeInstanceOf(Object);
+
+    expect(response.body).toEqual({ comment: "small cat" });
+  });
```

Comment:
Ideally need to check the whole object has come back - you're going to be using this returned object to update the front end, so you want to make sure you've got the full thing - .toMatchObject

Reply:
Done.